lib/rules/selector-max-attribute/README.md

```diff
@@ -0,0 +1,77 @@
+# selector-max-attribute
+
+Limit the number of attribute selectors in a selector.
+
+```css
+    [rel="external"] {}
+/** ↑
+ * This type of selector */
+```
+
+This rule resolves nested selectors before counting the number of attribute selectors. Each selector in a [selector list](https://www.w3.org/TR/selectors4/#selector-list) is evaluated separately.
+
+The `:not()` pseudo-class is also evaluated separately. The rule processes the argument as if it were an independent selector, and the result does not count toward the total for the entire selector.
+
+## Options
+
+`int`: Maximum attribute selectors allowed.
+
+For example, with `2`:
+
+The following patterns are considered warnings:
```

Comment:
Oops, didn't spot this. I had forgotten about this change :)

@evilebottnawi Please change to: "The following patterns are considered violations:"